src/components/popovers/Popover.jsx

```diff
@@ -157,26 +157,27 @@ const Popover = ({
   const clonedTrigger = React.cloneElement(trigger, triggerProps);
 
   // https://popper.js.org/popper-documentation.html#modifiers
-  const popperModifiers = {
-    offset: {
+  const popperModifiers = [
+    {
+      name: 'offset',
       enabled: true,
-      offset: `0,${distance}`,
+      options: {
+        offset: `0,${distance}`,
```

Comment:
This should be a tuple

Reply:
Thanks @FezVrasta, that seems to fix most of the breakage!